torch/fx/graph_module.py

```diff
@@ -0,0 +1,459 @@
+import torch
```

Comment:
The PR comment is great documentation for this module, can you put it in the docstrings for `torch.fx` so we keep it around?